.github/workflows/build.yaml

```diff
@@ -30,6 +30,11 @@ jobs:
       - name: license headers
         run: |
           cargo test -p kamu-repo-tools -- license_header
+      # TODO: Remove error suppression
+      #       https://github.com/kamu-data/kamu-cli/issues/332
+      - name: clippy
+        run: |
+          cargo clippy --all --all-targets -- -D warnings || true
```

Comment:
I took your branch and ran this command, and it produced no warnings or errors. If that's what you see too, what prevents us from enabling this lint?

Reply:
> I took your branch and ran this command, and it produced no warnings or errors. If that's what you see too, what prevents us from enabling this lint?

hm, I just made a plan to include enabling a little later, as I expected a lot of warnings/errors.
There were indeed a lot of bugs, but they've all been fixed -- so I don't see any barriers to inclusion.

Will do

Reply:
Done